packages/editable/src/editable.tsx

```diff
@@ -160,6 +161,41 @@ if (__DEV__) {
   EditableInput.displayName = "EditableInput"
 }
 
+export interface EditableTextareaProps extends HTMLChakraProps<"textarea"> {}
+
+/**
+ * EditableTextarea
+ *
+ * The textarea used in the `edit` mode
+ */
+export const EditableTextarea = forwardRef<EditableTextareaProps, "textarea">(
+  (props, ref) => {
+    const { getTextareaProps } = useEditableContext()
+    const styles = useStyleConfig("Textarea", props)
+
+    const textareaProps = getTextareaProps(
+      props,
+      ref,
+    ) as HTMLChakraProps<"textarea">
```

Comment:
Can we move the type casting to the function `getTextareaProps`?
I think this should simplify the usage and the type safety of `getTextareaProps` and `getInputProps`.

e.g.

```ts
function getTextareaProps(): HTMLChakraProps<"textarea"> {}
function getInputProps(): HTMLChakraProps<"input"> {}
```

or if TS yells at you

```ts
function getTextareaProps() {
  return ({}) as HTMLChakraProps<"textarea">
}

Reply:
Thank you for your good idea!
I think you idea is reasonable enough to make change on it.
I may work on it next week.
Thanks again!

Reply:
I made changes on new commit!
I added default HTMLTypes on PropGetter!